<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoBlendWidget</class>
 <widget class="QWidget" name="AutoBlendWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>118</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#AutoBlendWidget{background:transparent}

QPushButton { 
	background:rgb(90,90,90,128);
	border: 3px solid rgb(90,90,90,0);
	padding: 5px;
    color: white;
	font-size: 12px;
	text-align:left;
}

QPushButton:hover{ background:rgb(255,153,0); border-color:rgb(255,153,0); }
QPushButton:pressed{ background:rgb(255,153,0); border-color:#FFB444; top:2px; }
QPushButton:checked{ background:rgb(255,153,0); border-color:#FFFFFF; top:2px; }

QLabel{color:white;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="blendButton">
     <property name="text">
      <string>Blend</string>
     </property>
     <property name="icon">
      <iconset resource="../../media/TopoBlender.qrc">
       <normaloff>:/icons/ManualBlend/mb_blend.png</normaloff>:/icons/ManualBlend/mb_blend.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="categoriesBox"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPushButton" name="analyzeButton">
     <property name="text">
      <string>Analyze...</string>
     </property>
     <property name="icon">
      <iconset resource="../../media/TopoBlender.qrc">
       <normaloff>:/icons/ManualBlend/mb_analyze.png</normaloff>:/icons/ManualBlend/mb_analyze.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Details</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="levelDetails">
     <item>
      <property name="text">
       <string>Low</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>High</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Count</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="count">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>512</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../media/TopoBlender.qrc"/>
 </resources>
 <connections/>
</ui>
